tests/e2e/e2e_grpc_secure_origination_test.go

```diff
@@ -45,16 +45,18 @@ func testSecureGRPCTraffic() {
 		}
 
 		// Get simple pod definitions for the gRPC server
-		svcAccDef, podDef, svcDef := Td.SimplePodApp(
+		svcAccDef, podDef, svcDef, err := Td.SimplePodApp(
 			SimplePodAppDef{
 				Name:        destName,
 				Namespace:   destName,
 				Image:       "moul/grpcbin",
 				Ports:       []int{grpcbinSecurePort},
 				AppProtocol: "tcp",
+				OS:          Td.ClusterOS,
```

Comment:
This should be hardcoded to be "linux" since the image is hardcoded as well. The image and the nodeSelector should be coupled

Reply:
This is being set as the linux OS constant elsewhere